src/front/glsl/functions.rs

```diff
@@ -583,7 +586,7 @@ impl Program<'_> {
                 let idx = arguments.len() as u32;
 
                 arguments.push(FunctionArgument {
-                    name: None,
+                    name: arg.name.clone(),
```

Comment:
This is actually significant, as the entry point function argument names are picked up by the reflection as binding names. The user code that matches the bindings by name expects those to be not changed. This is why the generated global names are also modified above, so the namer doesn't insert extra "1" at the end due to name collision.

Reply:
This seems like it could easily break a better option would be to return some kind of reflection data but I believe this should be handled by the backends not the frontends

Reply:
Right now this is what reflection looks at, and it's important at least for bevy to have that work. I'm OK with revisiting that later if we get the reflection data back in some other way, but given how non-intrusive this change is, I'd like to keep it for now.